spk/duplicity/Makefile

```diff
@@ -0,0 +1,43 @@
+SPK_NAME = duplicity
+SPK_VERS = 0.7.03
+SPK_REV = 1
+SPK_ICON = src/duplicity.png
+DSM_UI_DIR = app
+
+BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel cross/$(SPK_NAME)
+WHEELS = src/requirements.txt
+SPK_DEPENDS = "python>=2.7"
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Encrypted bandwidth-efficient backup using the rsync algorithm.
+RELOAD_UI = yes
```

Comment:
If there's no UI, and package is not startable via GUI (i.e. command line only), there's no need for this.